configs/config.ci.yaml

```diff
@@ -79,7 +79,6 @@ app:
   validate_api: true
   # main secret for various MAC and encryptions like domain registration
   # token and encrypted private JWKs. "random" generates an ephemeral secret.
-  secret: random
+  secret: 1w8KZbew7DzhxKKOY7O_cgVnyVWCl5dGp78uaLoxgbg
```

Comment:
How did you generate this secret?

Reply:
by running `python -c "import secrets; print(secrets.token_urlsafe())"`